packages/@glimmer/integration-tests/lib/suites/has-block.ts

```diff
@@ -262,4 +262,48 @@ export class HasBlockSuite extends RenderTest {
     this.assertComponent('', { 'data-has-block': 'false' });
     this.assertStableRerender();
   }
+
+  @test({ kind: 'glimmer' })
+  'has-block works within a yielded curried component invoked as variable'() {
```

Comment:
"invoked as a variable" doesn't really make sense to me in this context. Maybe:

```suggestion
  'has-block works within a yielded curried component invoked within mustaches'() {
```